test.js

```diff
@@ -135,4 +135,29 @@ describe('notify(opts)', () => {
 			''
 		].join('\n'));
 	});
+
+	it('should exclude -g argument when `isGlobal` option is `false`', () => {
+		function Control() {
+			this.packageName = 'update-notifier-tester';
+			this.update = {
+				current: '0.0.2',
+				latest: '1.0.0'
+			};
+		}
+		util.inherits(Control, updateNotifier.UpdateNotifier);
```

Comment:
This is now duplicated. Can you make it out of these test function and to the top of the function.

Bonus points for making it a ES2015 class instead.